test/helpers.go

```diff
@@ -0,0 +1,89 @@
+package test
```

Comment:
We already have a package for this kind of thing. It's in `internal/test`.
I would suggest putting this file there until we make the decision to move everything here/somewhere else.